stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -457,6 +457,14 @@ function(_compile_swift_files
     list(APPEND swift_flags "-Xfrontend" "-sil-verify-all")
   endif()
 
+  if(SWIFT_SIL_VERIFY_ALL_MACOS_ONLY)
```

Comment:
A bit torn here if we should fold this `if` block in the previous one -- both for clarity and to ensure the flags gets added once at most